understanding/20/contrast-minimum.html

```diff
@@ -8,6 +8,16 @@
 <body>
    <h1>Understanding Contrast (Minimum)</h1>
    
+   <section id="brief">
+		<h2>In brief</h2>
+		<dl>
+		<dt>Objective</dt><dd>Text can be seen by more people</dd>
+		<dt>Author task</dt><dd>Sufficiently contrast text against its background</dd>
+		<dt>Key beneficiaries</dt><dd>Users with low vision</dd>
```

Comment:
@gundulaniemann wrote
> there are various vision impairments where users rely on good contrast.
what about:
> Key beneficiaries
Users with vision impairments
I'd also agree to "Users with reduced vision"

"Low vision" has been used as the catch-all for this group until now (e.g., Low Vision Task Force). Open to change based AGWG support. Note that the EOWG is recommending removing the "Key beneficiaries" line since they will be adding significantly more content to the User Benefits sections as well as to the Introduction document. See https://github.com/w3c/wai-intro-wcag/issues/197

Reply:
Is the term 'low vision' defined somewhere?

Reply:
Not in the standard, although the phrase is in the abstract and a requirement note. Might be something in https://www.w3.org/WAI/GL/task-forces/low-vision-a11y-tf/

In any event, marking this resolved since the beneficiaries material was removed.